functions/accept.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+const AWS = require("aws-sdk");
+const S3 = new AWS.S3({ apiVersion: "2006-03-01" });
+const SQS = new AWS.SQS({ apiVersion: "2012-11-05" });
+
+const { CONFIG_TABLE, QUEUE_NAME, LOCKS_TABLE, CONTENT_BUCKET } = process.env;
+
+module.exports.handler = async function(
+  {
+    resource,
+    path,
+    body,
+    headers,
+    queryStringParameters,
+    requestContext: {
+      stage,
+      requestTimeEpoch,
+      httpMethod,
+      requestId
+    }
+  },
+  context
+) {
+  console.time("accept");
+  const responseCode = 200;
+  const responseBody = { requestId };
+
+  console.time("acceptS3");
+  const result = await S3.putObject({

```

Comment:
Fake "image" content saved to S3 here, just to ensure I'm able to access it on the message processing side